jobs/ci-kubernetes-e2e-gce-taint-evict.env

```diff
@@ -0,0 +1,15 @@
+# GCE ENV
+KUBERNETES_PROVIDER=gce
+KUBE_GCE_ZONE=us-central1-f
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### job-env
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:TaintEviction\]
+PROJECT=k8s-jkns-gce-taintevict
+# TODO: Enable this when we've split 1.2 tests into another project.
+FAIL_ON_GCP_RESOURCE_LEAK=false
+KUBE_NODE_OS_DISTRIBUTION=gci
+
+CONTROLLER_MANAGER_TEST_ARGS="${CONTROLLER_MANAGER_TEST_ARGS:-} --use-taint-based-evictions=true --enable-taint-manager=true"
```

Comment:
env file does not know how to evaluate things like ${CONTROLLER_MANAGER_TEST_ARGS:-}. If you want to run it locally just set it locally, it will overwrite the env var.

And add an entry to https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml :-)